language/de/extimgaslink.php

```diff
@@ -23,5 +23,11 @@
 }
 
 $lang = array_merge($lang, array(
+	'EXTIMGASLINK_CONFIG'			=> 'Erlaubte Herkunft von in Beiträgen verwendeten Bildern',
+	'EXTIMGASLINK_CONFIG_EXPLAIN'	=> 'Alle anderen Bilder werden durch Links ersetzt.',
+	'EXTIMGASLINK_INVALID_CONFIG'	=> 'Die angegebene Bildherkunft ist ungültig.',
+	'EXTIMGASLINK_SECURE_SITES'		=> 'nur Bilder von sicheren Webseiten und dem Server',
+	'EXTIMGASLINK_SERVER_ONLY'		=> 'nur Bilder vom Server',
```

Comment:
Ich denke schon. "anderen" könntest du ggf. noch durch “übrigen“ ersetzen, das passt glaube ich besser zum Stil der de Übersetzung, aber das geht auch so noch in Ordnung.  

Für die beiden anderen Sprachen gilt natürlich das gleiche.